contracts/LPVault.sol

```diff
@@ -42,45 +62,50 @@ contract LPVault is Owned {
-    ///  this modifier
+    /// @dev The attached `LiquidPledging` contract is the only address that can
+    ///  call a function with this modifier
     modifier onlyLiquidPledging() {
         require(msg.sender == address(liquidPledging));
         _;
     }
-    /// @dev USED FOR TESTING???
+    /// @dev Used for testing... SHOULD BE REMOVED FOR MAINNET LAUNCH OR MAYBE NOT????????????????????....................
     function VaultMock() public pure {
 
     }
-
+    /// @dev The fall back function allows ETH to be deposited into the LPVault
+    ///  through a simple send
     function () public payable {
 
     }
 
-    /// @notice `setLiquidPledging` is used to attach a specific liquid pledging
-    ///  instance to this LPvault. Keep in mind this isn't a single pledge but
-    ///  instead an entire liquid pledging contract.
+    /// @notice `onlyOwner` used to attach a specific liquidPledging instance
+    ///  to this LPvault; keep in mind that once a liquidPledging contract is 
+    ///  attached it cannot be undone, this vault will be forever connected
     /// @param _newLiquidPledging A full liquid pledging contract
     function setLiquidPledging(address _newLiquidPledging) public onlyOwner {
         require(address(liquidPledging) == 0x0);
         liquidPledging = LiquidPledging(_newLiquidPledging);
+        VaultSet(_newLiquidPledging);
     }
 
-    /// @notice `setAutopay` is used to toggle whether the LPvault will
-    ///  automatically confirm a payment after the payment has been authorized.
-    /// @param _automatic If true payments will confirm automatically
+    /// @notice Used to decentralize, toggles whether the LPVault will
+    ///  automatically confirm a payment after the payment has been authorized
+    /// @param _automatic If true, payments will confirm instantly, if false
+    ///  the training wheels are put on and the owner must manually approve 
+    ///  every payment
     function setAutopay(bool _automatic) public onlyOwner {
         autoPay = _automatic;
+        AutoPaySet();
     }
 
-    /// @notice `authorizePayment` is used in order to approve a payment 
-    ///  from the liquid pledging contract. Whenever a project or other address
-    ///  needs to receive a payment it needs to be authorized with this contract.
-    /// @param _ref This parameter is used to reference details about the
-    ///  payment from another contract.
-    /// @param _dest This is the address that payments will end up being sent to
-    /// @param _amount This is the amount that the payment is being authorized
-    ///  for.
+    /// @notice `onlyLiquidPledging` authorizes payments from this contract, if 
+    ///  `autoPay == true` the transfer happens automatically `else` the `owner`
+    ///  must call `confirmPayment()` for a transfer to occur (training wheels);
+    ///  either way, a new payment is added to `payments[]` 
+    /// @param _ref References the payment can be a URL, IPFS hash, etc
```

Comment:
hmm, well what ever is used here will be an argument to `liquidPledging.cancelPayment` and `liquidPledging.confirmPayment` call in the `doConfirmPayment` and `doCancelPayment` vault methods. Since this is the `LPVault` and meant to work with `liquidPledging`, this will be the `pledgeId`